Files/MultilingualResources/Files.zh-Hans.xlf

```diff
@@ -1092,27 +1092,27 @@
         </trans-unit>
         <trans-unit id="RenameError.ItemDeleted.Text" translate="yes" xml:space="preserve">
           <source>The item you're attempting to rename no longer exists. Please verify the correct location of the item you need to rename.</source>
-          <target state="new">The item you're attempting to rename no longer exists. Please verify the correct location of the item you need to rename.</target>
+          <target state="new">你正在尝试重命名的项目不存在。请确认你需要重命名项目的正确位置。</target>
```

Comment:
All states of translated items should be changed to "translated"

Reply:
Got it!